CSharpToVB/TextProgressBar.vb

```diff
@@ -55,4 +53,14 @@ Public Class TextProgressBar
         End With
     End Sub
 
+    Public Sub Report(value As ProgressReport) Implements IProgress(Of ProgressReport).Report
+        With _progressBar
+            .Visible = value.Maximum = 0 OrElse value.Current < value.Maximum
+            .Maximum = value.Maximum
+            .Value = value.Current
+        End With
+
+        pbPrecentage(_progressBar, $"{value.Current:N0} of {value.Maximum:N0}")
+    End Sub
```

Comment:
I didn't add Thread.Sleep because I didn't quite understand the purpose. Is the above code okay, or should I add it somewhere in here?